vcr/interface.go

```diff
@@ -54,6 +54,12 @@ var vcDocumentType = "application/vc+json"
 
 var revocationDocumentType = "application/vc+json;type=revocation"
 
+// ConceptFinder can resolve VC backed concepts for a DID.
+type ConceptFinder interface {
+	// Find returns the requested concept or ErrNotFound
+	Find(conceptName string, ID did.DID) (concept.Concept, error)
```

Comment:
In our NutsOrganizationCredential ID is a DID, but it could be very well something else. We could make the API broader usable by accepting ID as string (also since it's converted to a string anyways before handing it to the database layer).

Reply:
actually it must be a URI: https://www.w3.org/TR/vc-data-model/#identifiers